tox.ini

```diff
@@ -164,7 +165,7 @@ changedir = docs
 commands =
   sphinx-build -E -a -W -b html -T . _build/html
 
-[testenv:py38-tracecontext]
```

Comment:
why change this?

Reply:
We want the `pytest-benchmark` tests to run together with the `instrumentation`, `core`, and `exporter` tests (the `build` workflow). But we don't want them to run with the `mypy` `mypyinstalled` and `tracecontext` tests.

Those latter tests were a special case in the matrix that only ever ran on Python 3.8.

So I moved the latter tests to the `misc` workflow and that way I only have to add the step to `Record the benchmarks` to the `build` workflow. However, the `misc` workflow doesn't check different Python version (it should check different versions). That's why `tracecontext`, `mypy` and `mypyinstalled` were a good candidate for the `misc` workflow because they also do not check version.

So `tox.ini` was expecting `py38-tracecontext` because of the `build` workflow matrix but now it will only receive `tracecontext` from the `misc` workflow.

Reply:
i.e. here: https://github.com/open-telemetry/opentelemetry-python/pull/1443/files/513e9ed265bf725c8e224c5dbc6cc7f3c4f2f2c7#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R88